src/GitHub.Exports.Reactive/Extensions/ConnectionManagerExtensions.cs

```diff
@@ -13,5 +14,13 @@ public static IObservable<bool> IsLoggedIn(this IConnectionManager cm, IReposito
                     .SelectMany(c => c.Login())
                     .Select(c => hosts.LookupHost(c.HostAddress)).Any(h => h.IsLoggedIn);
         }
+
+        public static IObservable<bool> IsLoggedIn(this IConnectionManager cm, IRepositoryHosts hosts, HostAddress address)
+        {
+            return cm.Connections.ToObservable()
+                    .Where(c => c.HostAddress.Equals(address))
+                    .SelectMany(c => c.Login())
+                    .Select(c => hosts.LookupHost(c.HostAddress)).Any(h => h.IsLoggedIn);
```

Comment:
I think putting the `.Any(...)` call here on a new line aligned with `.Where` `.SelectMany` and `Select` would be more consistent with how we chain methods elsewhere.